controllers/topics.controllers.js

```diff
@@ -1,8 +1,17 @@
-const { fetchTopics } = require('../models/topics.models');
+const { fetchTopics, fetchArticleById } = require('../models/topics.models');
 
 exports.getTopics = (req, res, next) => {
     fetchTopics().then((topics) => {
         res.status(200).send({topics});
     })
     .catch(next);
+}
+
+exports.getArticleById = (req, res, next) => {
+    const id = req.params.article_id;
+    fetchArticleById(id)
+    .then((article) => {
+        res.status(200).send(article);
```

Comment:
Just make sure to send data in an object

So it would be `res.status(200).send({article})`

Makes life a lot easier on the front end!